### ISSUE TITLE
Create new comunication chanel - discourse

### ISSUE THREAD

**Issue by SebastianWesolowski** [xFAANG/askql#119]
https://www.discourse.org/ is a self-service hub of common questions, interesting ideas, and useful discussion.

I saw this idea on https://discourse.roots.io

They use discourse for user whos use their tools. But github issue only for bug, task and devloping project

**Comment by mhagmajer** [comment]
I like the idea of creating a Q&A platform for the community. Discourse is cool. Let's compare different options and their respective impact for the growing AskQL community.

**Stack Overflow**
Known and widely used platform for questions and answers related to programming. Someone [with at least 1500 reputation could create a tag for AskQL](https://stackoverflow.com/help/privileges/create-tags) for us to use. Stack Overflow doesn't require any maintenance on our side. This would also help with the SEO and to further spread the word about this technology.

**Comment by czerwinskilukasz1** [comment]
@mhagmajer , I would say Stack Overflow is a great idea and we should definitely answer questions that people ask on AskQL there. _However_, wouldn't it be nice to have a dedicated forum for users anyway?

**Comment by mhagmajer** [comment]
Yes, dedicated forum is nice and I'm just listing options at this point. I think that whatever platform we choose, it would be more efficient for the community if there is just one that we officially support and recommend. I proposed Stack Overflow because with tags it can provide a pleasant scoped environment - see Meteor for example https://stackoverflow.com/questions/tagged/meteor

**Comment by SebastianWesolowski** [comment]
@mhagmajer, I'm sure Stack Owerflow has a better infrastructure and question and answer tool.

But my experience with discourse is easier to introduce, and start writing in small group. 

Additional profit from such a solution is easier move user to main page (discourse.askql.org -> askql.org)

Of course it only my thinking. I don't insist on it tool ;)

**Comment by mhagmajer** [comment]
Seems like GitHub and Discord currently works out with our scales. All new comers can ask questions using GitHub issues. If this doesn't work for as then we can revisit this discussion - Discourse vs SO vs anything else.

**Comment by czerwinskilukasz1** [comment]
Agreed.

**Comment by mhagmajer** [comment]
We should however make it more encouraging for people to ask us question with GitHub issues. Maybe a direct link from askql.org and/or README.md to ask a question by posting an issue?
